src/tests/ftest/cart/test_group_np_common.h

```diff
@@ -187,6 +193,8 @@ test_swim_status_handler(crt_rpc_t *rpc_req)
 		  e_req->rank, swim_seq,
 		  e_req->exp_status);
 
+	D_FREE(swim_seq);
```

Comment:
This is still incorrect. You can't free  swim_seq where you do because it introduces another bug. If swim_seq is assigned "" you're leaking swim_seq and then trying to free an invalid address.

Reply:
That assignment happens on line 163.

Reply:
6adad54f11d227290cdf2423bc118ce29cc5dc14